source/puppet/4.10/lang_data_number.md

```diff
@@ -57,49 +56,63 @@ $product = 8 * 3e5  # evaluates to 240000.0
 
 Integer values can be expressed in decimal notation (base 10), octal notation (base 8), and hexadecimal notation (base 16).
 
-* _Non-zero_ decimal integers must not start with a `0`.
-* Octal values have a prefix of `0`, which can be followed by a sequence of octal digits 0-7.
-* Hexadecimal values have a prefix of `0x` or `0X`, which can be followed by hexadecimal digits 0-9, a-f, or A-F.
+-   _Non-zero_ decimal integers must not start with a `0`.
+-   Octal values have a prefix of `0`, which can be followed by a sequence of octal digits 0-7.
+-   Hexadecimal values have a prefix of `0x` or `0X`, which can be followed by hexadecimal digits 0-9, a-f, or A-F.
 
 Floats can't be expressed in octal or hex.
```

Comment:
This is wrong. All examples below (lines 66 to 76) are about integers. We don't allow floats expressed as octal and hex